Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -183,9 +183,9 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['TTbar_13_ID']=1
             wmsplit['SingleMuPt10FS_ID']=1
             wmsplit['TTbarFS_ID']=1
-            wmsplit['RECODR2_50nsreHLT']=1
-            wmsplit['RECODR2_25nsreHLT']=1
-            wmsplit['RECODR2_2016sreHLT']=1
+            wmsplit['RECODR2_50nsreHLT']=5
+            wmsplit['RECODR2_25nsreHLT']=5
+            wmsplit['RECODR2_2016reHLT']=5
```

Comment:
its beginning to look like these things should become properties of the workflows themselves rather than having a long list in MatrixInjector (which should know much about specific workflows)
@hengne